### DISCUSSION TITLE
[禁止登录]当前上网环境异常，请更换网络环境或在常用设备上登录或稍后再试。

### DISCUSSION THREAD

**Discussion by ghost** [howmanybots/onebot#144] [问答]
ticket也输入了，同一台电脑这个电脑QQ客户端可以正常登录

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/63281264/105368586-ab7fba80-5c3c-11eb-8a12-2cb2009f8bcb.png)

**Comment by ieew** [comment]
这个问题与OneBot无关，并且mirai方面有解决方法，请去mirai查阅相关文档

**Comment by ieew** [comment]
> > @sam01101 这个问题与OneBot无关，并且mirai方面有解决方法，请去mirai查阅相关文档
> 
> 回错人了（

第一次使用tgbot，尴尬
